main.go

```diff
@@ -1,23 +1,28 @@
 package main
 
 import (
+	"errors"
 	"flag"
 	"os"
 
 	"github.com/v3io/sidecar-proxy/app"
+	"github.com/v3io/sidecar-proxy/app/metrics"
 
 	"github.com/sirupsen/logrus"
 )
 
 func main() {
 
+	var metricNames app.StringSliceFlag
+
 	// args
 	listenAddress := flag.String("listen-addr", os.Getenv("PROXY_LISTEN_ADDRESS"), "Port to listen on")
 	forwardAddress := flag.String("forward-addr", os.Getenv("PROXY_FORWARD_ADDRESS"), "IP /w port to forward to (without protocol)")
 	namespace := flag.String("namespace", os.Getenv("PROXY_NAMESPACE"), "Kubernetes namespace")
 	serviceName := flag.String("service-name", os.Getenv("PROXY_SERVICE_NAME"), "Service which the proxy serves")
 	instanceName := flag.String("instance-name", os.Getenv("PROXY_INSTANCE_NAME"), "Deployment instance name")
 	logLevel := flag.String("log-level", os.Getenv("LOG_LEVEL"), "Set proxy's log level")
+	flag.Var(&metricNames, "metric-names", "Set which metrics to collect")
```

Comment:
Should be `metric-name` because of the usage (the flag is given per metric name, multiple times)